tracks/csharp/exercises/space-age/mentoring.md

```diff
@@ -0,0 +1,86 @@
+### Reasonable solutions
+
+#### using `Dictionary`
+
+```csharp
+using System.Collections.Generic;
+
+public class SpaceAge
+{
+    private readonly double earthYear;
+    private const double OrbitalPeriodEarth = 31_557_600.0;
+
+    private static Dictionary<string, double> conversionFactor 
+            = new Dictionary<string, double>()
```

Comment:
Mayeb put this on the same line as the declaration?

Reply:
sure, done